state-chain/custom-rpc/src/lib.rs

```diff
@@ -1839,24 +1782,21 @@ where
 			.await
 	}
 
-	fn cf_lp_get_order_fills(
-		&self,
-		at: Option<state_chain_runtime::Hash>,
-	) -> RpcResult<BlockUpdate<OrderFills>> {
+	fn cf_lp_get_order_fills(&self, at: Option<Hash>) -> RpcResult<BlockUpdate<OrderFills>> {
 		order_fills::order_fills_for_block(
 			self.rpc_backend.client.as_ref(),
 			at.unwrap_or_else(|| self.rpc_backend.client.info().finalized_hash),
 		)
 	}
 
-	fn cf_supported_assets(&self) -> RpcResult<Vec<Asset>> {
+	fn cf_supported_assets(&self, _at: Option<Hash>) -> RpcResult<Vec<Asset>> {
```

Comment:
We've added the param but we ignore it - in fact based on how this is implemented (ie. in the rpc layer), we can't support this for past runtime versions... We should push the implementation into a runtime rpc.